style-sheets/primevue-components-overrides.scss

```diff
@@ -1140,4 +1140,133 @@ label {
         display: block;
         margin-top: 1.5rem;
     }
+}
+
+/* ------------------------------------ tabView ----------------------------- */
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-panels {
+    padding: 0;
+    background-color: transparent;
+    z-index: 1;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav {
+    border: none;
+    position: absolute;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header {
+    border-bottom: #ffffff;
+    display: flex;
```

Comment:
I don't think we need this at all, and you need to check all your other styles, if really need them, we have to make it clear and easy to understand so we can maintain for long term. Many of the "display: flex" I don't think we need.